### ISSUE TITLE
scss template doesn't produce valid scss

### ISSUE THREAD

**Issue by rlahoda** [liquidlight/sass-gulp-svg-sprite#4]
I have this set up in my gulpfile to export the _sprite.scss file but when the file is processed by postcss i get the error that it isn't valid css. 

I checked with the developer for gulp-svg-sprite and it turned out that the mustache template I copied off this repo didn't produce valid scss. I'm not sure if your code is more sophisticated than I'm familiar with but I thought I'd let you know this was happening.

Aside from that, the blogpost was really helpful and explained everything very well so thanks for your work on that!

Here's the error:

```
events.js:141
      throw er; // Unhandled 'error' event
      ^
Error: dev/scss/modules/_sprite.scss
Error: (sprite: (width: 147px, height: 129px, pngPath: "../img/sprite.png", svgPath: "../img/sprite.svg"), casicon: (width: 32.88px, height: 22.19px, backgroundX: 0px, backgroundY: 0px), facebook: (width: 10px, height: 22px, backgroundX: -43px, backgroundY: -33px), twitter: (width: 32px, height: 22px, backgroundX: -63px, backgroundY: -65px), twitterHover: (width: 32px, height: 22px, backgroundX: -105px, backgroundY: -97px)) isn't a valid CSS value.
        on line 2 of dev/scss/modules/_sprite.scss
>>     sprite: (width: 147px, height: 129px, pngPath: '../img/sprite.png', svgPat
   ----^

    at options.error (/web_projects/sandbox_homepage/node_modules/gulp-sass/node_modules/node-sass/lib/index.js:271:32)
```

Here's the _sprite.scss file that's generated:

```
$icons: (
    sprite: (width: 147px, height: 129px, pngPath: '../assets/img/sprite.png', svgPath: '..../assets/img/sprite.svg'),
    casicon: (width: 32.88px, height: 22.189999999999998px, backgroundX: 0px, backgroundY: 0px),
    facebook: (width: 10px, height: 22px, backgroundX: -43px, backgroundY: -33px),
    twitter: (width: 32px, height: 22px, backgroundX: -63px, backgroundY: -65px),
    twitterHover: (width: 32px, height: 22px, backgroundX: -105px, backgroundY: -97px),
);
```

**Comment by escapedcat** [comment]
You are using the `scss` file directly with postcss? I that supposed to work? Usually I use compiled `css` from a `scss` file with postcss. Or are you using [a postcss-scss plugin](https://github.com/postcss/postcss-scss)?

Sorry if I get you wrong here. Just wanted to clarify.

**Comment by rlahoda** [comment]
Sorry about that, I got a little mixed up looking at my gulpfile. It's not running directly through postcss, the scss is actually being processed by gulp-sass then postcss is running an autoprefixer on it.

**Comment by mikestreety** [comment]
@rlahoda Is this now resolved? Glad the post and repo helped 👍 

If not, would you mind posting your gulpfile?

**Comment by rlahoda** [comment]
Ok, so I had to step away from this and do other work so this morning I came back and decided to go back through everything step-by-step to make sure i didn't miss something. somehow that mysteriously fixed whatever was going on. Thinking about it, it makes sense. there's no reason that the code from the icon map should be reaching postcss since it's entirely a sass item so i don't know how it was getting pulled into that in the first place. i pasted in the same code from before, changed the paths to the ones that would work for my project and it worked. i don't get it.
